### DISCUSSION TITLE
Adding a Chat Widget Option

### DISCUSSION THREAD

**Discussion by gqoew** [open-webui/open-webui#3835] [Ideas]
It would be amazing to have an open webui "chat widget" option, a collapsible chat window fixed on the bottom right of a web page that would allow users to start a conversation directly from a web page

**Comment by flefevre** [comment]
This chat widget should take as input a model assistant (or a set of model assistants) with their knowledge base.
So you can talk with a referetial.
Those chat widget should have the same privacy option, public or group private

**Comment by flefevre** [comment]
Any news on this feature?
It is provided by concurrent projects

**Comment by mig5** [comment]
I would love to see this. I started my 'self hosted AI/LLM' journey by looking at Gurubase, but right when I went to use it, they closed their source code, so now I don't want to use it. I was hoping OpenWeb UI might have something.

Instead I've found AnythingLLM, but the quality of the responses, even when using the same backend Ollama model and the same provided knowledge base, doesn't seem as good as the OpenWeb UI responses in the web interface.

**Comment by MOEI-123412** [comment]
This feature would be so incredible useful!
